tests/Agent/IntegrationTests/Shared/MsSqlOdbcConfiguration.cs

```diff
@@ -3,6 +3,7 @@
 
 
 using System;
+using System.Collections.Generic;
```

Comment:
This can be removed since its not used.

Reply:
Thanks for noticing, my last push removed that.